default/stringtables/en.txt

```diff
@@ -10448,91 +10448,92 @@ SHP_WEAPON_1_2
 Mass Driver 2
 
 SHP_WEAPON_1_2_DESC
-Improves Mass Driver by one level, increasing strength by 1.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 4.
```

Comment:
This is the only comment where I'd like a bit of a debate before making changes.

Can you give an example where the later is not correct?
If you are referring to the species piloting, the ship part description adds that information (that there may be additional modifiers so that "base" damage can be different than "shot" damage).

The ship part lists "shot damage", which is always the value from level 1 of the weapon.
Saying that "increasing shot damage to X" matches up with the ship part description.

"Increasing by X" might be confusing in multiple ways.
One user said they couldn't always remember the initial number, so saying that it increases by X made him have to do extra work to get the number he wanted.
If Laser4 increases by 6, is that 6 more than Laser3, or 6 more than Laser1? Giving an absolute number removes that ambiguity.